bin/localTunnel.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+'use strict'
+
+const localtunnel = require('localtunnel')
+require('./normalizeEnv')
+
+const connect = (port, host) => {
+  const subdomain = host.replace(/\.localtunnel\.me$/, '')
+  try {
+    const tunnel = localtunnel(port, { subdomain }, (err, tunnel) => {
+      if (err) {
+        console.error('Error: Failed to create tunnel to ' + process.env.CLIENT_HOST + ':', err)
+        process.exit(1)
+      }
+
+      console.log('Created localtunnel to', tunnel.url)
+      tunnel.on('close', connect.bind(null, port, host))
+
+      // localtunnel sometimes crashes and has to be restarted
+      tunnel.on('error', connect.bind(null, port, host))
+    })
+  } catch (err) {
+    console.log('Error occurred:', err.message, 'Restarting localtunnel.')
+    connect(port, host)
+  }  
+}
+
+// if the hostname is a subdomain of "localtunnel.me"
+if (process.env.CLIENT_HOST.match(/\.localtunnel\.me$/)) {
+  connect(process.env.CLIENT_PORT, process.env.CLIENT_HOST)
+} else {
+  console.log('Host is not on localtunnel. Exiting with success.')
```

Comment:
But if I'm not using localtunnel (i.e. I've set up my own domain) I wouldn't expect anything to happen with localtunnel so I think it should exit silently.